apps/nx-aws-cache-iac/README.md

```diff
@@ -0,0 +1,39 @@
+# NX AWS Cache Infrastructure as Code
+
+Sets the infrastructure for the AWS Cache for NX. This is a cache that is shared between all the projects in the monorepo.
```

Comment:
For me it's still not clear what this package does, what is the purpose? And is it ok to be defined as an app and not a package in `packages` folder?

Reply:
can you let me know if is now more clear?